ci/jenkins/test-rancher.sh

```diff
@@ -236,9 +238,9 @@ function run_conformance {
 
     set +e
     if [[ "$TESTCASE" =~ "conformance" ]]; then
-        ${WORKSPACE}/ci/run-k8s-e2e-tests.sh --e2e-conformance --log-mode $MODE --image-pull-policy ${IMAGE_PULL_POLICY} --kubernetes-version "auto" > ${WORKSPACE}/test-result.log
+        ${WORKSPACE}/ci/run-k8s-e2e-tests.sh --e2e-conformance --log-mode $MODE --image-pull-policy ${IMAGE_PULL_POLICY} --kubernetes-version "auto" --rancher-testbed > ${WORKSPACE}/test-result.log
```

Comment:
Do we have a log rotation policy for this testbed ?, log rotation policy is a good practice to manage the growth of log files and to ensure that you have a consistent and manageable logging strategy for test-result.log. Log rotation typically involves periodically archiving old log files and deleting them after a certain period or when they reach a certain size, to prevent disk space from being exhausted.